src/Razor/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorTagHelperBinderPhase.cs

```diff
@@ -247,8 +247,8 @@ public ComponentDirectiveVisitor(string filePath, IReadOnlyList<TagHelperDescrip
                     {
                         // If this is a child content tag helper, we want to add it if it's original type is in scope.
                         // E.g, if the type name is `Test.MyComponent.ChildContent`, we want to add it if `Test.MyComponent` is in scope.
-                        TrySplitNamespaceAndType(typeName, out var typeNameTextSpan, out var _);
-                        typeName = GetTextSpanContent(typeNameTextSpan, typeName);
+                        TrySplitNamespaceAndType(typeName, out var namespaceTextSpan, out var _);
```

Comment:
Are these just renames?

Reply:
Yep to match the signature https://github.com/dotnet/aspnetcore/pull/23627/files/5ed18a194f7d0ddd70b14d9deb1c6f8fc76c97c5#diff-e1c88de1d8233078ce6bb0bc46a2aab7R418